Dockerfile_cc_for_arm

```diff
@@ -25,6 +25,12 @@ RUN apt install -y \
     g++-aarch64-linux-gnu \
     g++-arm-linux-gnueabihf
 
+# Install gxx-6 compiler and toolchain for RPi
+RUN apt-get install -y \
+  gcc-6 \
+  g++-6 \
+  g++-6-arm-linux-gnueabihf
```

Comment:
Is there a reason to pick GCC version 6 instead of GCC 7.3 that comes as default in Bionic?

Reply:
Ok, just saw the comment on the other PR related to the GCC version and issue with Poco.

Reply:
Yes exactly, that's the only reason. Honestly I don't know the differences between the two versions of GCC, but with this one and adding the CMAKE compiler flags, it's able to find Poco.
This is probably an issue of the `arm-linux-gnueabihf` compiler, because when you are not cross-compiling it always works out of the box

Reply:
Does this solve the poco problems for other arm targets or only for Raspbian? If it is only for Raspbian, these changes should probably be added to the Raspbian Docker file.

Reply:
See comment https://github.com/ros2/cross_compile/pull/15#issuecomment-490825415